src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.jsx

```diff
@@ -86,12 +86,26 @@ export default class SentryApplicationDetails extends AsyncView {
     browserHistory.push(`/settings/${orgId}/developer-settings/`);
   };
 
+  onFieldChange = (name, value) => {
```

Comment:
@EvanPurkhiser Is this OK? Or is this function too hacky?

Reply:
cc: the form master @billyvg

Reply:
Yeah this should be fine